### PR TITLE
fix: en typo (taht -> that)

### PR DESCRIPTION
**Pull request by chuganzy**
I am not sure if this is intentional or not though😅

| Before | After |
| --- | --- |
| Screen Shot 2022-03-19 at 12 27 34 AM | Screen Shot 2022-03-19 at 12 27 22 AM |

### CODE REVIEW

File: locales/en/messages.po [hunk 1]
```diff
@@ -68,7 +68,7 @@ msgstr "2:20 PM"
 
 # the typo here is intentional - please maintain a similar unprofessional tone!
```
Comment (notjosh) [inline locales/en/messages.po:69]: Definitely intentional!

**Comment by notjosh** [comment]
Thanks for noticing! But you're right to wonder - he typo is definitely intentional, and trying to convey an unprofessional tone in the "bad example".

File: locales/en/messages.po (hunk 1 above)
Comment (chuganzy) [reply locales/en/messages.po:69]: Ugh! I missed this comment! 🙏

**Comment by chuganzy** [comment]
Lol! Got it! Makes sense!

**Pull request closed without merging**